exonum-java-binding-core/src/test/java/com/exonum/binding/proxy/NativeHandleTest.java

```diff
@@ -53,6 +51,7 @@ void close() {
 
     nativeHandle.close();
     assertFalse(nativeHandle.isValid());
+    assertThat(nativeHandle.toString()).contains(convertHandle(handle));
```

Comment:
Nit: I'd suggest using a string so that the expected format is apparent and this code is more readable: "11" or "0x11" — whatever it is.

Reply:
I'll create String constant for this.